onmt/Models.py

```diff
@@ -53,34 +53,54 @@ def forward(self, input, lengths=None, hidden=None):
 class RNNEncoder(EncoderBase):
     """ The standard RNN encoder. """
     def __init__(self, rnn_type, bidirectional, num_layers,
-                 hidden_size, dropout, embeddings):
+                 hidden_size, no_pack_padded_seq,
```

Comment:
This doesn't need to be an option that users know about. Just have it be set automatically if they pass SRU